### ISSUE TITLE
Crash in rav1e::common::parse_config unless --min_quantizer is passed

### ISSUE THREAD

**Issue by jbeich** [xiph/rav1e#1350]
Regressed by #1348.

```
$ rustc -vV
rustc 1.37.0-nightly
binary: rustc
commit-hash: 991c719a1d0f95c37ed7ea56bdb38bcc2a6246b9
commit-date: 2019-06-08
host: x86_64-unknown-freebsd
release: 1.37.0-nightly
LLVM version: 8.0
```

```
$ RUST_BACKTRACE=1 cargo run foo.y4m -o foo.ivf
    Finished dev [optimized + debuginfo] target(s) in 0.08s
     Running `target/debug/rav1e foo.y4m -o foo.ivf`
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:347:21
stack backtrace:
   0: std::sys_common::backtrace::print
   1: std::panicking::default_hook::{{closure}}
   2: std::panicking::default_hook
   3: std::panicking::rust_panic_with_hook
   4: std::panicking::continue_panic_fmt
   5: rust_begin_unwind
   6: core::panicking::panic_fmt
   7: core::panicking::panic
   8: core::option::Option<T>::unwrap
             at .../rustc-nightly-src/src/libcore/macros.rs:12
   9: rav1e::common::parse_config
             at src/bin/common.rs:454
  10: rav1e::common::parse_cli
             at src/bin/common.rs:327
  11: rav1e::main
             at src/bin/rav1e.rs:144
  12: std::rt::lang_start::{{closure}}
             at .../rustc-nightly-src/src/libstd/rt.rs:64
  13: std::panicking::try::do_call
  14: __rust_maybe_catch_panic
  15: std::rt::lang_start_internal
  16: main
  17: _start
             at /usr/src/lib/csu/amd64/crt1.c:76
```

**Comment by lu-zero** [comment]
Thank you for reporting, https://github.com/xiph/rav1e/pull/1351 should address it.

**Comment by dwbuiten** [comment]
Fixed in #1351.
